definitions/npm/express_v4.x.x/flow_>=v0.25.x/express_v4.x.x.js

```diff
@@ -196,9 +196,11 @@ declare module 'express' {
     use(middleware: Middleware): this;
     use(...middelware: Array<Middleware>): this;
     use(path: string, ...middelware: Array<Middleware>): this;
+    use(path: string, router: Router): this;
 
     route(path: string): Route;
   }
+  declare type $Router = Router;
```

Comment:
Because `Router` is a class, it's already possible to use `import type` with it -- so no need to define a special type export here.

(Same goes for `Application`/`$Application` now that I'm looking. I didn't notice this in the original libdef)